docs/docs/concepts/command.md

```diff
@@ -6,7 +6,7 @@ Commands form the API of your domain. Like [Events](event.html), they are simple
 Commands have descriptive names describing the intent of what you are trying to achieve, for example `CreateUser` or `SendInvoice`.
 Commands inherit from `Sequent::Command`. Additionally
 you can add [Validations](validations.html) to commands to ensure correctness. Sequent uses
-[ActiveModel::Validations](http://api.rubyonrails.org/classes/ActiveModel/Validations.html)
+[ActiveModel::Validations](http://api.rubyonrails.org/classes/ActiveModel/Validations.html){:target="_blank"}
```

Comment:
I don't like opening a new page by default, people can use cmd-click for this and leave it up to them.

Reply:
I'm sorry, I don't agree. I think it's nice to have links on the same website open in the same window, while external links open in a new tab. This way users stay at the website and don't accidentally click away never to come back.

Reply:
Sure, you can do that by cmd click. Especially on mobile it is annoying when a new tab/window opens. But as this highly depends on the user we should keep it optional. By enforcing a new window, we enforce everyone to use it like this.